atd-vzd/migrations/default/1715094801442_add_import_log_table/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE cris_import_log (
+    id SERIAL PRIMARY KEY,
+    object_path TEXT NOT NULL,
+    object_name TEXT NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    completed_at TIMESTAMPTZ DEFAULT NULL,
+    records_processed JSONB DEFAULT '{}',
```

Comment:
Yes, this is really Frank writing this. I think this is a good place for a JSON blob in lieu of a one-to-many table FK relationship because I want the log to fit neatly in one table. I'm seeing this field getting populated with a dict that has the keys of the CRIS schema year and the value being the number of crashes which we processed. I think if we develop that metadata to be any more complex, we should bail out of the JSONB field and switch to proper DB tables with normal, normal relationships.